File: WebApp/WebApp/Models/Review.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.Models
{
    public class Review
    {
        public int Id { get; set; }
        [Required]
        public User User { get; set; }
        [Required]
        public Flight Flight { get; set; }
        [Range(1, 5)]
        public int Rating { get; set; }
    }
}

File: angular-app/src/models/Review.ts
import { User } from './User';
import { Flight } from './Flight';

export class Review {
    constructor(userId: number, flightId: number, rating: number){
        this.UserId = userId,
        this.FlightId = flightId,
        this.Rating = rating
    }

    Id: number;
    UserId: number;
    FlightId: number;
    Rating: number;
}